src/agents/graph.py

```
@@ -0,0 +1,157 @@
+            "awaiting_confirmation": False,
+            "extracted_data": {},
+            "query_results": [],
+            "error_message": None,
+            "retry_count": 0,
+            "agent_data": {},
+            "timestamp": datetime.utcnow().isoformat(),
+            "session_metadata": {}
+        }
+        
+        # Configuration for checkpointing
+        config = {
```

Ruff (0.12.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)